api/server/server.go

```diff
@@ -1080,9 +1080,13 @@ func (s *Server) postContainersStop(eng *engine.Engine, version version.Version,
 		return fmt.Errorf("Missing parameter")
 	}
 
-	seconds, err := strconv.Atoi(r.Form.Get("t"))
-	if err != nil {
-		return err
+	seconds := 0
+	if len(r.Form.Get("t")) > 0 {
+		secs, err := strconv.Atoi(r.Form.Get("t"))
```

Comment:
would you like me to change it like below?
        seconds, err := strconv.Atoi(r.Form.Get("t"))
        if err != nil {
                if err.(*strconv.NumError).Err == strconv.ErrSyntax {
                        seconds = 0
                } else {
                        return err
                }
        }